Incomplete symbol is not fixed up by importing a package

### The following program `sample.go` triggers an unexpected result

```go
func TestIssueX(t *testing.T) {
	i := interp.New(interp.Options{Env: []string{"foo=bar"}})
	err := i.Use(stdlib.Symbols)
	if err != nil {
		t.Fatal(err)
	}

	_, err = i.Eval(`x := errors.New("")`)
	if err == nil {
		t.Fatal("Expected an error")
	}

	_, err = i.Eval(`import "errors"`)
	if err != nil {
		t.Fatal(err)
	}

	_, err = i.Eval(`x := errors.New("")`)
	if err != nil {
		t.Fatal(err)
	}
}
```


### Expected result

```console
$ go test
=== RUN   TestIssueX
Hello
--- PASS: TestIssueX (0.00s)
PASS
ok      github.com/traefik/yaegi/interp 0.007s
```


### Got

```console
$ go test
=== RUN   TestIssueX
    /home/firelizzard/src/yaegi-fork/interp/interp_eval_test.go:1767: <nil>
--- FAIL: TestIssueX (0.00s)
FAIL
FAIL	github.com/traefik/yaegi/interp	0.007s
```


### Yaegi Version

5665c9a4108e5f878c0d1d8d1e1913eb3c314154

### Additional Notes

_No response_

--- a/interp/gta.go
+++ b/interp/gta.go
@@ -217,6 +217,12 @@ func (interp *Interpreter) gta(root *node, rpath, importPath, pkgName string) ([
 					if name == "" {
 						name = interp.pkgNames[ipath]
 					}
+
+					// If an incomplete type exists, delete it
+					if sym, exists := sc.sym[name]; exists && sym.kind == typeSym && sym.typ.incomplete {
+						delete(sc.sym, name)
+					}
+
 					// Imports of a same package are all mapped in the same scope, so we cannot just
 					// map them by their names, otherwise we could have collisions from same-name
 					// imports in different source files of the same package. Therefore, we suffix

--- a/interp/interp_eval_test.go
+++ b/interp/interp_eval_test.go
@@ -1769,7 +1769,7 @@ func TestIssue1388(t *testing.T) {
 	}
 
 	_, err = i.Eval(`x := errors.New("")`)
-  	if err != nil {
+	if err != nil {
 		t.Fatal(err)
 	}
 }

--- a/interp/interp_eval_test.go
+++ b/interp/interp_eval_test.go
@@ -1749,3 +1749,26 @@ func TestRestrictedEnv(t *testing.T) {
 		t.Fatal("expected \"\", got " + s)
 	}
 }
+
+func TestIssue1388(t *testing.T) {
+	i := interp.New(interp.Options{Env: []string{"foo=bar"}})
+	err := i.Use(stdlib.Symbols)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = i.Eval(`x := errors.New("")`)
+	if err == nil {
+		t.Fatal("Expected an error")
+	}
+
+	_, err = i.Eval(`import "errors"`)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = i.Eval(`x := errors.New("")`)
+	if err != nil {
+		t.Fatal(err)
+	}
+}

